lib/msal-node/src/cache/Storage.ts

```diff
@@ -245,7 +245,8 @@ export class Storage extends CacheManager {
 
         // read all keys
         Object.keys(cache).forEach(key => {
-            Object.keys(key).forEach(internalKey => {
+            // @ts-ignore
```

Comment:
Dead code here and line 266

Reply:
It isn't dead code but rather syntax for `//ts-ignore`. This part of the code has changed and no longer applicable.